pkg/api/exec.go

```diff
@@ -345,3 +355,75 @@ func (c *ExecController) handleOutput(ctx context.Context, cmt *github.Comment,
 	}
 	return nil
 }
+
+func (c *ExecController) setUpdatedCommentID(ctx context.Context, cmt *github.Comment, updateCondition string) error { //nolint:funlen
```

Comment:
This method is almost identical to post command, so I think refactoring it other PR.